lib/profile/commands/configure.rb

```diff
@@ -54,7 +70,16 @@ def prompt
       end
 
       def cluster_type
-        @type ||= Type.find( prompt.select('Cluster type: ', Type.all.map { |t| t.name }) )
+        return @type if @type
```

Comment:
Good spot (also the return statement was leftover from some modifications I was making that are since reverted, I understand that `||=` does the same thing)